Tasks/PowerShellOnTargetMachines2/task.json

```diff
@@ -0,0 +1,201 @@
+        },
+        {
+            "name": "UserPassword",
+            "type": "string",
+            "label": "Password",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Administrator password for the target machines. <br>It can accept variable defined in Build/Release definitions as '$(passwordVariable)'. <br>You may mark variable type as 'secret' to secure it."
+        },
+        {
+            "name": "ScriptType",
+            "type": "radio",
+            "label": "Script Type",
+            "defaultValue": "FilePath",
+            "options": {
+                "FilePath": "File Path",
+                "Inline": "Inline"
+            },
+            "helpMarkDown": "The type of script to execute: Inline or File Path",
+            "groupName": "ScriptOptions"
+        },
+        {
+            "name": "ScriptPath",
+            "type": "string",
+            "label": "Script File Path",
+            "defaultValue": "",
+            "visibleRule": "ScriptType = FilePath",
+            "required": true,
+            "helpMarkDown": "Location of the PowerShell script on the target machines or on a UNC path like C:\\BudgetIT\\Web\\Deploy\\Website.ps1",
+            "groupName": "ScriptOptions"
+        },
+        {
+            "name": "InlineScript",
+            "type": "multiLine",
+            "label": "Script",
+            "visibleRule": "ScriptType = Inline",
+            "required": true,
+            "defaultValue": "# Write your powershell commands here.\n\nWrite-Output \"Hello World\"",
+            "properties": {
+                "resizable": "true",
+                "rows": "10",
+                "maxLength": "5000"
+            },
+            "helpMarkDown": "",
+            "groupName": "ScriptOptions"
+        },
+        {
+            "name": "ScriptArguments",
+            "type": "string",
+            "label": "Script Arguments",
+            "defaultValue": "",
+            "required": false,
+            "properties": {
+                "editorExtension": "ms.vss-services-azure.parameters-grid"
+            },
+            "helpMarkDown": "Arguments for the PowerShell script. Can be ordinal parameters or named parameters like -testParam test",
+            "groupName": "ScriptOptions"
+        },
+        {
+            "name": "Protocol",
+            "type": "radio",
+            "label": "Protocol",
+            "required": false,
+            "defaultValue": "Https",
+            "options": {
+                "Http": "HTTP",
+                "Https": "HTTPS"
+            },
+            "helpMarkDown": "Select the protocol to use for the WinRM connection with the machine(s). Default is HTTPS.",
+            "groupName": "SessionOptions"
+        },
+        {
+            "name": "NewPsSessionOptionArguments",
+            "type": "multiLine",
+            "label": "New-PSSessionOption arguments",
+            "required": false,
+            "defaultValue": "",
```

Comment:
We should fill the default with "-skipCA"

Reply:
yes. this is a part of the next PR's